### ISSUE TITLE
Add option to hide stats

### ISSUE THREAD

**Issue by stimfrog** [duolicious/duolicious-backend#508]
Gives Replies To: 100% of Intros and Gets replies to: 0.001 % of Intros is kinda bad looking.

I don't have an issue with the other stats.

**Comment by duogenesis** [comment]
> Gets replies to: 0.001 % of Intros is kinda bad looking

That's true. It's intended to encourage message senders to think about whether the recipient would want to get that message. Although users have immutable attributes (e.g. gender) which affect their reply rate, they still have a remarkable amount of control over how many replies they get, as explained in [this](https://duolicious.app/blog/how-to-get-more-replies-on-dating-apps/) blog.

Still, including the "gets replies" percentage on people's profiles might not actually improve how thoughtful the average user is when sending messages. It might just make respectful users too uncomfortable to send messages, while allowing the brazen users to continue spamming without a second thought. I need to leave this feature in-place for a while so that I can measure how showing the reply rates affects them in the long run.

Also, now that you bring it up, I just realized that someone's reply rate would immediately drop to 0% after sending their very first intro. So the "gets replies" statistic should probably only be shown after there's been enough time to collect a big enough sample of sent intros to be accurate. I've raised a ticket for this issue: https://github.com/duolicious/duolicious-backend/issues/509

**Comment by stimfrog** [comment]
Fair enough
